common/src/main/scala/explore/common/UserPreferencesQueries.scala

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.target
+
+import clue.GraphQLOperation
+import clue.macros.GraphQL
+import explore.GraphQLSchemas.UserPreferencesDB
+
+object UserPreferencesQueries {
+
+  @GraphQL
+  object UserInsertMutation extends GraphQLOperation[UserPreferencesDB] {
+    val document = """
+      mutation insert_user($id: String) {
+        insert_lucuma_user_one(
+          object: {
+            user_id: $id
+          },
+          on_conflict: {
+            update_columns: [],
+            constraint: lucuma_user_pkey
+          }
+        ) {
+          user_id
+          }
+        }
+    """
+  }
+
+  @GraphQL
+  object UserWidthsCreation extends GraphQLOperation[UserPreferencesDB] {
+    val document = """
+        mutation initial_area_width($userId: String, $width: Int) {
```

Comment:
If I understand correctly, I maybe able to replace this by a single `WidthInsertInput` field though I'm not yet able to do so.

Reply:
Does it mean the document is sent when the macro is executed?

Reply:
When the query is executed.